packages/botonic-plugin-contentful/src/contentful/manage/manage-contentful.ts

```diff
@@ -60,17 +60,27 @@ export class ManageContentful implements ManageCms {
     context: ManageContext,
     contentId: ContentId,
     fieldType: ContentFieldType,
-    value: any
+    value: any,
+    code?: string
   ): Promise<void> {
     const environment = await this.getEnvironment()
     const oldEntry = await environment.getEntry(contentId.id)
+
     const field = this.checkOverwrite(context, oldEntry, fieldType, true)
     oldEntry.fields[field.cmsName][context.locale] = value
```

Comment:
code is not localizable, so this might not work.
Maybe you should have a isFieldLocalizable function and the here assign directly (without [context.locale]) for codes.

Reply:
This line is for replacing the content of the ContentFieldType selected and this is localizable. The replace of the Code is done in `replaceCode` method (next line)